module.nix

```
@@ -121,7 +121,7 @@ in
           enable = true;
           port = 6379;
         };
-        postgresql = {
+        postgresql = mkIf cfg.createDatabase {
           enable = true;
           package = pkgs.postgresql_14;
           ensureDatabases = mkIf cfg.createDatabase [ "authentik" ];
```

These inner `mkIf cfg.createDatabase` can be removed now that all of the postgresql service config is behind this check above:
```suggestion
          ensureDatabases = [ "authentik" ];
```
(also the one below, but I'm unable to reference it in this review suggestion :))

---

done